### PR TITLE
Added availability request with tests and docs.

### PR DESCRIPTION
**Pull request by shmuga**

### CODE REVIEW

File: src/Services/Air/AirParser.js [hunk 1]
```diff
@@ -994,6 +994,60 @@ function exchangeBooking(rsp) {
...
+
+    leg.push(s);
+
+    if (!isConnected) {
+      results.push(leg);
+      leg = [];
+    }
+  });
+
+  if (leg.length !== 0) {
```
Comment (dchertousov) [inline src/Services/Air/AirParser.js]: When `leg.length` could equal `0`?
Is it an error to throw?

File: src/Services/Air/AirParser.js [hunk 2]
```diff
@@ -994,6 +994,60 @@ function exchangeBooking(rsp) {
...
+    results.push(leg);
+  }
+
+  return {
+    legs: results,
+    nextResultReference: rsp[`common_${this.uapi_version}:NextResultReference`],
```
Comment (dchertousov) [inline src/Services/Air/AirParser.js]: You return reference but you never use it in requests. Shouldn't it be added as a param?

File: src/Services/Air/AirParser.js (hunk 1 above)
Comment (shmuga) [reply src/Services/Air/AirParser.js]: No, It can be caused when last segment is connection.

File: src/Services/Air/AirParser.js (hunk 2 above)
Comment (shmuga) [reply src/Services/Air/AirParser.js]: Yes, I missed this.

**Pull request merged**